WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -154,11 +161,11 @@ private extension CartView {
             Button {
                 cartViewModel.addMoreToCart()
             } label: {
-                Image(PointOfSaleAssets.cartBack.imageName)
-                    .resizable()
-                    .frame(width: 32, height: 32)
+                Image(systemName: Constants.backButtonSymbol)
+                    .frame(width: Constants.backButtonSize, height: Constants.backButtonSize)
+                    .font(Font.posTitle)
```

Comment:
I think the button can be a bit smaller
<img width="1269" alt="Smaller icon" src="https://github.com/user-attachments/assets/89c9194a-93f9-42a1-b59a-7ff84e6d9f26">

From the designs, it uses 24px bold which is `posBodyEmphasized`